torch/testing/_internal/opinfo/core.py

```diff
@@ -1213,7 +1213,7 @@ def get_inplace_operator(self):
     # in a way that works like it should, and I tried a LOT of things.
     def _maybe_skip_or_xfail(self, rules, device, sample, idx):
         def _subtest_fn(test_case, sample=sample, idx=idx):
-            return test_case.subTest(sample=sample, idx=idx)
+            return test_case.subTest(sample=sample.name, idx=idx)
```

Comment:
I guess there's no issues with capturing sample in `_subtest_fn()`? To be extra safe I'd probably capture sample.name there too lol